build.gradle

```diff
@@ -158,7 +158,7 @@ test {
 dependencies {
     api platform('com.exactpro.th2:bom:3.0.0')
     api "com.exactpro.th2:cradle-core:${cradleVersion}"
-    api 'com.exactpro.th2:grpc-common:3.8.0'
+    api 'com.exactpro.th2:grpc-common:3.8.0-th2-2567-1380894854-SNAPSHOT'
```

Comment:
gRPC version will be changed after https://github.com/th2-net/th2-grpc-common/pull/34